src/lib/detection_algorithm_manager/main.js

```diff
@@ -0,0 +1,42 @@
+//read from config to find detection algorthims and paths
+const path = require('path');
+const config_path = path.join(__dirname, 'config.json')
+const config = require(config_path);
+
+//load detection modules
+var detection_modules = {}
+Object.keys(config).forEach((key) => {
+    detection_modules[key] =  require(path.join(__dirname, config[key]));
+});
+
+function DetectionAlgorithmManager() {
+    var self = this;
+    //this returns a list of the availble detection modules
+    this.getAvailbleAlgorthims = function getAvailbleAlgorthims() {
+        return Object.keys(config);
+    },
+
+    //Set the  exporter to the specified detection module
+    this.initExporter = function(algorithm, exportDirPath, classes, posFramesCount, frameWidth, frameHeight, testSplit, cb) {
+         if (!Object.keys(config).includes(algorithm)){
+             throw (`Error ${algorithm} module is not recognized`);
+         }
+         var exporter = new detection_modules[algorithm].Exporter(exportDirPath, classes, posFramesCount, frameWidth, frameHeight, testSplit);
+         exporter.init().then(()=> {
+             return cb(null, exporter.exportFrame);         
+         }).catch((err) =>{
+            return cb(err);
+         });
+    },
+
+    this.initReviewer = function (algorithm, modelPath, cb) {
```

Comment:
You can just have a single reviewImagesFolder method here instead of initReviewer (since you call it only once), but then again I'm fine with the current interface as well since we might move at some point to "review single image"